### DISCUSSION TITLE
How to set Access-Headers to public assets directory access

### DISCUSSION THREAD

**Discussion by takutope** [nitrojs/nitro#346] [Q&A]
Nuxt3 SSR production build creates some mjs and css files in `.output/public/_nuxt/` directory.
I would like to set access header  `"Cache-Control": "max-age=31536000"` for static assets access via nitro.

I tried config like below, but it does not work.

```
 {
    publicAssets: [
      {
        maxAge: 31536000,
      }
    ]
}
```

Is there any solution?

**Comment by itsmnthn** [comment]
I've been trying this config on nitropack api and it is not working for me too

**Comment by jschroeter** [comment]
for me it works with this:

```
export default defineNuxtConfig({
  routeRules: {
    '/_nuxt/**': { headers: { 'cache-control': 'max-age=31536000,immutable' } }
  }
})
```

Actually I think this should be the default for the _nuxt directory.
